scripts/install/RegisterTestRunnerPlugins.php

```diff
@@ -38,41 +36,33 @@ class RegisterTestRunnerPlugins extends InstallAction
     public static $plugins = [
         'security' => [
             [
-                'id' => 'sectionPause',
-                'name' => 'Section Pause',
-                'module' => 'taoTestRunnerPlugins/runner/plugins/security/sectionPause',
+                'id' => 'blurPause',
```

Comment:
Could you please let the 2 plugins `sectionPause` and `autoPause` registered but inactive.